services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/model.tsx

```diff
@@ -19,11 +19,22 @@ const Model = ({ match }) => {
     queryFn: fetchModel
   });
 
-  // function handleSetModelNotes(text: string){ setModelNotes(text) };
+  const initialNoteValue = !modelData ? '' : modelData.model.notes;
+  const [modelNotes, setModelNotes] = React.useState<string>(initialNoteValue);
 
-  // function onHandleUpdateModel() {
+  function handleSetModelNotes(text: string){ setModelNotes(text) };
 
-  // }
+  function onHandleUpdateModel() {
+    request.put({
+      url: `${process.env.DEFAULT_URL}/api/v1/comprehension/automl_models/${modelId}`,
+      json: { automl_model: {notes: modelNotes} }, },
+      (error, httpStatus, body) => {
+        if (httpStatus.statusCode === 200 || httpStatus.statusCode === 204) {
+          return
+        }
+        setErrors({status: httpStatus.statusCode, message: httpStatus.statusMessage})
+      });
+  }
```

Comment:
I think Eric mentioned this, but we've been following a pattern where the request is made in a shared file.

Reply:
addressed